src/utils/SignedMessage.js

```diff
@@ -0,0 +1,67 @@
+import React, {useEffect, useState} from "react";
+import {observer} from "mobx-react";
+import {Link, Redirect, useRouteMatch} from "react-router-dom";
+import {checkoutStore, rootStore} from "Stores";
+import UrlJoin from "url-join";
+import ImageIcon from "Components/common/ImageIcon";
+
+import {PageLoader} from "Components/common/Loaders";
+import {ButtonWithLoader, LocalizeString} from "Components/common/UIComponents";
+import ListingIcon from "Assets/icons/listings icon";
+
+import BackIcon from "Assets/icons/arrow-left";
+import Confirm from "Components/common/Confirm";
+import NFTCard from "Components/nft/NFTCard";
+import {LoginGate} from "Components/common/LoginGate";
+
+const { ElvClient } = require("@eluvio/elv-client-js");
+const Utils = require("@eluvio/elv-client-js/src/Utils");
+
+// Create a signed JSON message
+const CreateSignedMessageJSON = async ({
```

Comment:
for record - signing code will be moved to client
https://github.com/eluv-io/elv-client-js/pull/267#event-12093183969

Reply:
using that now